spec/controllers/posts_controller_spec.rb

```diff
@@ -1,11 +1,12 @@
 require 'spec_helper'
+require 'pry'
```

Comment:
I've been avoiding to make `pry` a project's dependency, even in development env, but I think this decision isn't actually such a big deal. Anyway, let's try to require `pry` only when we really need to debug things thus avoiding extra loading of code. Can you just remove this line?